src/governance-object.h

```diff
@@ -25,6 +25,7 @@ class CGovernanceVote;
 
 static const int MAX_GOVERNANCE_OBJECT_DATA_SIZE = 16 * 1024;
 static const int MIN_GOVERNANCE_PEER_PROTO_VERSION = 70204;
+static const int GOVERNANCE_FILTER_PROTO_VERSION = 70206;
```

Comment:
All new const (introduced in this PR) and MIN_GOVERNANCE_PEER_PROTO_VERSION aren't used anywhere "lower" than governance.cpp in hierarchy and they have nothing to do with single object as far as I see, more for managing multiple objects.